sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -587,7 +630,7 @@ export class MessageSender extends LinkEntity {
         this.name,
         data
       );
-      return await this._trySend(encodedMessage);
+      return await this._trySend(encodedMessage, false, options);
```

Comment:
Ah, the curse of no longer being the last default parameter ;)

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8159#issuecomment-608254608